app/controllers/application_controller.rb

```diff
@@ -1,3 +1,18 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+  before_action :check_updates
+
+  def check_updates
+    # current_time = Time.now
+    # last_time = Item.last.updated_at
+    # if last_time.nil? || (last_time - current_time) > -20
+      update_companies_and_items
+    # end
+  end
```

Comment:
Why keep these comments? You can always check previous commits to get the code back.

Additionally maybe something like this would be a bit clearer:
```ruby
def check_updates
  update_companies_and_items if time_to_update?
end
```